parser/parser.y

```diff
@@ -560,6 +561,15 @@ AlterSpecification:
 KeyOrIndex:
 	"KEY"|"INDEX"
 
+ColumnOptColumnName:
+	"COLUMN"
+|	"COLUMN" "COLUMN"
```

Comment:
AFAICT, the original grammar (before this PR) would have accepted the sentence `drop column column` as a valid `AlterSpecification`. The first `column` instance is the optional unreserved `column` keyword, the second `column` instance is an `Identifier` and the real name of the column to drop.

That's the mess created by the keyword/identifiers mixing.